docs/install/tools-for-managing-visual-studio-instances.md

```diff
@@ -33,25 +33,31 @@ In addition, the [Setup Configuration API](<xref:Microsoft.VisualStudio.Setup.Co
 
 ## Using vswhere.exe
 
-`vswhere.exe` is automatically included in Visual Studio 2017 version 15.2 or later, or you can download it from [the releases page](https://github.com/Microsoft/vswhere/releases). Use `vswhere -?` to get help information about the tool. As an example, this command shows all releases of Visual Studio, including earlier versions of the product and prereleases, and outputs the results in JSON format:
+`vswhere.exe` is automatically included in Visual Studio (starting with Visual Studio 2017 version 15.2 and later versions), or you can download it from [the VSWhere releases page](https://github.com/Microsoft/vswhere/releases). Use `vswhere -?` to get help information about the tool. As an example, this command shows all releases of Visual Studio, including earlier versions of the product and prereleases, and outputs the results in JSON format:
```

Comment:
We never capitalize "vswhere". This is inconsistent with existing branding.

Reply:
@TerryGLee Can you fix the capitalization here?

Reply:
Yes; I have PR #[4378](https://github.com/MicrosoftDocs/visualstudio-docs-pr/pull/4378) in flight.